starter-code/main.js

```diff
@@ -0,0 +1,159 @@
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              }
+            });
+            break;
+          case "2":
+            db.collection('companies').find({}, {
+              name: 1,
+              _id: 0
+            }).count((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              }
+            });
+            break;
+          case "3":
+            db.collection('companies').find({
+              "founded_year": 2004
+            }).count((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              }
+            });
+            break;
+          case "4":
+            db.collection('companies').find({
+              $and: [{
+                "founded_year": 2004
+              }, {
+                "founded_month": 2
+              }]
+            }, {
+              name: 1,
+              _id: 0
+            }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              }
+            });
+            break;
+            case "5":
+              db.collection('companies').find({$and: [{"founded_year": 2004}, {$or:[{"founded_month": 4},{"founded_month": 5},{"founded_month": 6}]}]}, {
+                name: 1,
+                _id: 0
+              }).toArray((error, result) => {
+                if (error) {
+                  console.log(error);
+                  rl.question(`\nType enter to continue: `, (answer) => {
+                    mainMenu();
+                  });
+                } else {
+                  console.log(result);
+                  rl.question(`\nType enter to continue: `, (answer) => {
+                    mainMenu();
+                  });
+                }
```

Comment:
La quinta query mejor así, en lugar de ir mes por mes:

`case "5":
            db.collection('companies').find({
              "founded_year": 2004,
              "founded_month": {
                $gte: 4,
                $lte: 6
              }
            }, {
              name: 1,
              _id: 0
            }).sort({
              "founded_month": 1
            }).toArray((error, result) => {
              if (error) {
                console.log(error);
                rl.question(`\nType enter to continue: `, (answer) => {
                  mainMenu()
                });
              } else {
                console.log(result);
                rl.question(`\nType enter to continue: `, (answer) => {
                  mainMenu()
                });
              }
            })
            break;`